blocks/media-gallery/media-gallery.js

```diff
@@ -0,0 +1,32 @@
+import { createOptimizedPicture } from '../../scripts/aem.js';
+
+export default function decorate(block) {
+    // modify card structure to have a card-content div
+    for(let idx = 0; idx < block.children.length; ++idx) {
+        const child = block.children[idx];
+        child.classList.add('card');
+        child.classList.add(`card-${idx}`);
+
+        const cardContent = document.createElement('div');
+        cardContent.classList.add('card-content');
+
+        console.log(block.classList);
+        if(block.classList.contains('alternate-2-1')) {
+            const startButton = document.createElement('button');
+            startButton.classList.add('start-button');
```

Comment:
I wouldn't make this button dependent on the "alternate-2-1", but based on detection of a link to a video (for now i'd use https://mediaspace.esri.com/, maybe we can get confirmation as to whether everything there are videos or we need to add a explicit variant to the block).